src/window/empire.c

```diff
@@ -64,6 +64,9 @@
 #define MAX_TRADE_OPEN_BUTTONS 64
 #define MAX_SORTING_BUTTONS 64
 
+#define DEFAULT_DOT_SCALE 100//dot scales
```

Comment:
```suggestion
#define DEFAULT_DOT_SCALE 100
```

The comment is meaningless really. But the real thing that made me change this was that the lack of spacing between the number and the `//` was making me uncomfortable